packages/expo-media-library/src/MediaLibrary.ts

```diff
@@ -53,8 +53,19 @@ export type AssetInfo = Asset & {
   location?: Location;
   exif?: object;
   isFavorite?: boolean; //iOS only
+  isNetworkAsset?: boolean; //iOS only
 };
 
+export interface AssetInfoQueryOptions {
+  shouldDownloadFromNetwork?: boolean;
+}
+
+export interface AssetChangeEvent {
+  insertedAssets: Asset[];
+  deletedAssets: Asset[];
+  updatedAssets: Asset[];
```

Comment:
It should be optional, shouldn't it? cause on the Android, we don't return this field.

Reply:
Android platform itself does not have this event at all 🤔

Reply:
Ohh, you are right 💪